Dispatcher: use UIManager:sendEvent instead of the current instance (#7999)
koreader/koreader: doc/Events.md

```lua
widget_foo:handleEvent(Event:new("Timeout"))
```

> However if the widget might be destroyed during the event you can call @{ui.uimanager:sendEvent|UIManager:sendEvent} to propagate the event from the topmost widget or @{ui.uimanager:broadcastEvent|UIManager:broadcastEvent} to send the event to all widgets.

Reviewer: Technically UIManager, well, *sends* ^^ the event *to* the topmost widget with a matching handler. And, since widgets at that level are usually based on `WidgetContainer`, *that* widget then propagates it to its children.

Reviewer 2: @NiLuJe Something like this?
```suggestion
If the widget can be destroyed during the event you should call @{ui.uimanager:sendEvent|UIManager:sendEvent} to propagate the event from the topmost widget or @{ui.uimanager:broadcastEvent|UIManager:broadcastEvent} to send the event to all widgets with a matching handler.
```

Reviewer: Actually, just re-reading it now, I'm fine with the current wording ;p.

Reviewer 2: Alright, I'd still make these changes though. :-)
```suggestion
If the widget can be destroyed during the event you should call @{ui.uimanager:sendEvent|UIManager:sendEvent} to propagate the event from the topmost widget or @{ui.uimanager:broadcastEvent|UIManager:broadcastEvent} to send the event to all widgets.
```
